logicrunner/executionbroker.go

```diff
@@ -229,6 +234,14 @@ func (q *ExecutionBroker) finishTask(ctx context.Context, transcript *common.Tra
 	}
 }
 
+func (q *ExecutionBroker) cleanMutableQueue(ctx context.Context) {
+	q.stateLock.Lock()
+	defer q.stateLock.Unlock()
+
+	q.mutable.queue.Clean(ctx)
+	q.deduplicationTable = make(map[insolar.Reference]bool)
```

Comment:
this is wrong, deduplication table shouldn't be cleaned completely. delete only those you deleted from the queue